notebooks/8_combine_dataframes.ipynb

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
<p>I suggest moving the read_csv for <code>air_quality_stations</code> and <code>air_quality_parameters</code> further down the notebook until right before they are used. Currently, it's hard for me to follow the lesson because there's a lot of objects you show me right at the top, and not all of them are relevant yet, so I get distracted by what I need to pay attention to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>